CoffeeChat/Controller/MatchViewController.swift

```diff
@@ -28,29 +33,73 @@ class MatchViewController: UIViewController {
         MatchSummary(title: "He is also part of...", detail: "EzraBox")
     ]
 
+    init(hasReachedOut: Bool) {
+        self.hasReachedOut = hasReachedOut
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.navigationBar.isHidden = true
```

Comment:
Earlier you used 

    navigationController?.setNavigationBarHidden(false, animated: false)

I think you should stay consistent with how you're updating the navigation bar.